modules/ztp-adding-new-content-to-gitops-ztp.adoc

```diff
@@ -3,72 +3,151 @@
-+
-[source,text,subs="attributes+"]
+example
+└── policygentemplates
+    ├── dev.yaml
+    ├── kustomization.yaml
+    ├── mec-edge-sno1.yaml
+    ├── sno.yaml
+    └── source-crs <1>
+        ├── PaoCatalogSource.yaml
+        ├── PaoSubscription.yaml
+        ├── custom
+        |   ├── apiserver-config.yaml
+        |   └── disable-nic-lldp.yaml
+        └── elasticsearch
+            ├── ElasticsearchNS.yaml
+            └── ElasticsearchOperatorGroup.yaml
 ----
-FROM registry.redhat.io/openshift4/ztp-site-generate-rhel8:v{product-version}
+<1> The `source-crs` subdirectory is in the same directory as the `kustomization.yaml` file.
 
-ADD example-cr2.yaml /kustomize/plugin/ran.openshift.io/v1/policygentemplate/source-crs/
-ADD example-cr1.yaml /kustomize/plugin/ran.openshift.io/v1/policygentemplate/source-crs/
-----
+[IMPORTANT]
+====
+To use your own custom resources, ensure that the custom CR names differ from the default source CRs provided in the ZTP container.
+====
 
-. Open a terminal at the `ztp-update/` folder and rebuild the container:
+. Update the required `PolicyGenTemplate` CRs to include references to the content you added in the `source-crs/custom` directory.
 +
-[source,terminal,subs="attributes+"]
+[source,yaml]
 ----
-$ podman build -t ztp-site-generate-rhel8-custom:v{product-version}-custom-1
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: "group-dev"
+  namespace: "ztp-clusters"
+spec:
+  bindingRules:
+    dev: "true"
+  mcp: "master"
+  sourceFiles:
+    # These policies/CRs come from the internal container Image
+    #Cluster Logging
+    - fileName: ClusterLogNS.yaml
+      remediationAction: inform
+      policyName: "group-dev-cluster-log-ns"
+    - fileName: ClusterLogOperGroup.yaml
+      remediationAction: inform
+      policyName: "group-dev-cluster-log-operator-group"
+    - fileName: ClusterLogSubscription.yaml
+      remediationAction: inform
+      policyName: "group-dev-cluster-log-sub"
+    #Local Storage Operator
+    - fileName: StorageNS.yaml
+      remediationAction: inform
+      policyName: "group-dev-lso-ns"
+    - fileName: StorageOperGroup.yaml
+      remediationAction: inform
+      policyName: "group-dev-lso-operator-group"
+    - fileName: StorageSubscription.yaml
+      remediationAction: inform
+      policyName: "group-dev-lso-sub"
+    # Performance Addon Operator
+    - fileName: PaoSubscriptionNS.yaml
+      remediationAction: inform
+      policyName: "group-dev-pao-ns"
+    - fileName: PaoSubscriptionCatalogSource.yaml
+      remediationAction: inform
+      policyName: "group-dev-pao-cat-source"
+      spec:
+        image: <image URL here>
+    - fileName: PaoSubscription.yaml
+      remediationAction: inform
+      policyName: "group-dev-pao-sub"
+    #These are custom local polices that come from the source-crs directory in the git repo
+    #Elasticsearch Operator
+    - fileName: elasticsearch/ElasticsearchNS.yaml <1>
```

Comment:
Given that the callout description talks about the `custom` subdirectory, shouldn't this label be on either line 99 or 102?

Reply:
This actually raised an investigation on my part. This callout should be added to the `custom` subdir as well as `elasticsearch`. I'll update the PR with clarification